feeds/feed.go

```diff
@@ -9,11 +10,13 @@ import (
 
 const schemaVer = "1.0"
 
+var errCutoff = errors.New("package was created before cutoff time")
```

Comment:
Can this be const?

Reply:
I believe the convention is to use package level vars for wrapping errors with `Errorf` https://github.com/Djarvur/go-err113#reports